katana/katana.py

```diff
@@ -90,6 +93,207 @@ def readconfig():
     cfg = json.loads("".join(lines))
     return cfg
 
+@route('/searchkw', method='POST')
+def searchkw():
+    """
+    This method returns the ActionFile Path of the selected Driver.
+    """
+    value = parseString("".join(request.body))
+    indented_xml = "".join(value.toprettyxml(newl='\n'))
+    corrected_xml = remove_extra_newlines_char_xml(indented_xml)
+    corrected_xml = corrected_xml
+    with open('obj.txt', 'w') as f:
```

Comment:
Also regarding this comment:
If I launch katana outside of katana directory, it will try to create the out.txt file under my cwd, which has a chance of overwriting existed file